src/test/ui/rfc-2565-param-attrs/param-attrs-builtin-attrs.rs

```diff
@@ -0,0 +1,169 @@
+#![feature(param_attrs)]
+
+pub fn foo(#[inline] a: u32, #[test] b: i32) {}
+//~^ ERROR forbidden built-in attribute
+
+// self, &self and &mut self
+
+struct SelfStruct {}
+impl SelfStruct {
+    fn bar(
+        #[inline] a: i32,
+        //~^ ERROR forbidden built-in attribute
+        #[test] b: i32,
+        #[no_mangle] c: i32,
+        #[should_panic] d: i32,
+    ) {}
+}
+
+struct RefStruct {}
+impl RefStruct {
+    fn bar(
+        #[inline] a: i32,
```

Comment:
This seems to duplicate the above? (occurs below also?)

Reply:
Thanks. This was a mistake